src/fetchtastic/menu_app.py

```
@@ -0,0 +1,100 @@
+# src/fetchtastic/menu_app.py
+
+from typing import Any, Dict, Sequence, Union
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)